lib/class-wp-theme-json-gutenberg.php

```diff
@@ -2729,9 +2729,7 @@ public function get_root_layout_rules( $selector, $block_metadata ) {
 		$css .= '.wp-site-blocks > .alignright { float: right; margin-left: 2em; }';
 		$css .= '.wp-site-blocks > .aligncenter { justify-content: center; margin-left: auto; margin-right: auto; }';
 
-		$block_gap_value       = $this->theme_json['styles']['spacing']['blockGap'] ?? '0.5em';
-		$has_block_gap_support = isset( $this->theme_json['settings']['spacing']['blockGap'] );
-		if ( $has_block_gap_support ) {
+		if ( isset( $this->theme_json['settings']['spacing']['blockGap'] ) ) {
```

Comment:
Would it be worth adding a comment just before this line that block gap styles will be output if the value is anything other than `null`? Block gap is a bit different to other features in that the styles will still be output when the value is `false`, so I was wondering if it'd be helpful to be a bit more explicit about it here? (`$has_block_gap_support` sort of implicitly communicates that checking for `isset()` means that the support is enabled, whereas without the name of the variable, it might help to be a bit clearer about it).

Reply:
Sounds good to me. Thanks for the suggestion!